group-membership/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Scale access to external vendors and partners
+description: Share documents, events, and communications with users outside 
+your domain
+labels:  Sheets, Groups for Business, Apps Script
```

Comment:
IIUC you can just put 'Groups' instead of Groups for Business

Reply:
OK, updated as groups.